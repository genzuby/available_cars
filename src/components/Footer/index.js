import React from 'react';
import { COLOR } from '../../style/constant';

const footerStyle = {
  height: '80px',
  fontSize: '14px',
  color: COLOR.DARK_GRAY,
  borderTop: `2px solid ${COLOR.LIGHT_GRAY}`,
};

function Footer() {
  return (
    <footer
      className="container-fluid d-flex justify-content-center  align-items-center fixed-bottom p-4"
      style={footerStyle}
    >
      Â© AUTO1 Group 2018
    </footer>
  );
}

export default Footer;
