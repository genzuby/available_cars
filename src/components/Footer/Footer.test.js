import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Footer from './index';

it('render snapshot', () => {
  const { getByTestId } = render(<Footer />);
  expect(getByTestId('footer')).toMatchSnapshot();
});
