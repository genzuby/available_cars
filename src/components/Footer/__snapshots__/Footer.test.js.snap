// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render snapshot 1`] = `
<footer
  class="container-fluid d-flex justify-content-center  align-items-center p-4"
  data-testid="footer"
  style="height: 80px; font-size: 16px; color: rgb(74, 74, 74); border-top: 2px solid #EDEDED;"
>
  Â© AUTO1 Group 2018
</footer>
`;
